Gemfile

```diff
@@ -106,6 +106,7 @@ group :test do
   gem 'fuubar'
   gem 'machinist', '< 3'
   gem 'shoulda'
+  gem 'shoulda-matchers', '~> 4.0'
```

Comment:
You shouldn't need this dependency for any of the tests you added (remove from here and `Gemfile.lock`).